### ISSUE TITLE
Crunchyroll update

### ISSUE THREAD

**Issue by taki640** [grantwinney/hide-comments-everywhere#130]
The new Crunchyroll website was released so the url doesn't need "beta." anymore.
The element still the same.

```
crunchyroll\.com; #content > div > div.app-body-wrapper > div > div > div.content-wrapper--MF5LS > div > div.commenting-wrapper
```

**Comment by grantwinney** [comment]
Thanks for the heads up. I'll remove the beta one.

I checked out a couple news posts, and it looks like the elements I'm blocking are still working.

```css
.comment-more, div.comments
```

Is there a page where you're still seeing comments?

**Comment by taki640** [comment]
Yes, the watch page from any anime. [This one](https://www.crunchyroll.com/pt-br/watch/GPWUK0KV3/the-last-one) for example.

**Comment by grantwinney** [comment]
Thanks. Alright that should be blocked now, once you update the definitions on the options page. Let me know if you see anything else.

**Comment by taki640** [comment]
Ok! Thank you for your work. I love this extension.
